src/wix/WixToolset.Core/CommandLine/VersionCommand.cs

```diff
@@ -16,13 +16,6 @@ internal class VersionCommand : ICommandLineCommand
 
         public Task<int> ExecuteAsync(CancellationToken cancellationToken)
         {
-            // $(GitBaseVersionMajor).$(GitBaseVersionMinor).$(GitBaseVersionPatch)$(GitSemVerDashLabel)+$(Commit)
-            Console.WriteLine("{0}.{1}.{2}{3}+{4}", ThisAssembly.Git.BaseVersion.Major
-                                                  , ThisAssembly.Git.BaseVersion.Minor
-                                                  , ThisAssembly.Git.BaseVersion.Patch
-                                                  , ThisAssembly.Git.SemVer.DashLabel
-                                                  , ThisAssembly.Git.Commit);
-
```

Comment:
Isn't this removing the output of the `version` command? I think we still want this part, right?

Reply:
I don't think so -- The version is shown in the logo and since the version command explicitly ignores the -nologo switch it would be redundant:
```
C:\Solutions\OHI\Optimiser\WiX.2012>C:\Solutions\TemporarySolutions\wix4\build\wix\Release\publish\wix\wix --version
WiX Toolset Core version 4.0.0-preview.1-build.126+4d48856ec445e3ba2efa256e166f01455a859ff1
Copyright (c) .NET Foundation and contributors. All rights reserved.

Reply:
Sorry, let me be more clear. We want it. It is (always right now without `-nologo` support) redundant but we want it.

Fixing the issue does not require updating the `version` command at all.

Reply:
Would it be better to use the same version output as the logo instead of using the ThisAssembly.Git information?

Reply:
No. At this time, the `version` command always provides the full git version details. The `ProductVersion` will contain less information in "official" builds (for example the SHA hash is removed).

Reply:
Rob, sorry I missed this comment

I am a bit confused (as always? :-) ) but in my 'official build' here are the two outputs, PV is the ProductVersion info and GI is the github info version:
```
C:\Solutions\OHI\Optimiser\WiX.2012>C:\Solutions\TemporarySolutions\wix4\build\wix\Release\publish\wix\wix --version
WiX Toolset Core version 4.0.0-preview.1-build.162+0fb2a3c27c5ca58a28dae07053ad1411735b0889
Copyright (c) .NET Foundation and contributors. All rights reserved.
PV: 4.0.0-preview.1-build.162+0fb2a3c27c5ca58a28dae07053ad1411735b0889
GI: 4.0.0-preview.1-build.162+0fb2a3c2
```
I'll implement the GI version, just trying to understand your comment. When I run the 'build_official command I take it that the result is different then the release build?

Reply:
No worries. I missed your comment above mine for quite a while as well.

My memory of the official build is faulty. In `src\Directory.Build.targets` the `InformationalVersion` (aka: ProductVersion) is essentially the same as the value displayed in the `version` command. My thinking is still that the `version` command should display "all the information available at build time" and the header should display "whatever the product version of the file is". If/when those versions change then the code will remain correct.

My faulty memory is that I thought the ProductVersion already differed.